src/Uno.UWP/Storage/Helpers/StorageFile.skia.cs

```diff
@@ -0,0 +1,29 @@
+using System.Threading;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Windows.Storage.Helpers
+{
+	partial class StorageFileHelper
+	{
+		private static Task<bool> FileExistsInPackage(string filename)
+		{
+			var executingPath = Assembly.GetExecutingAssembly().Location;
+			if (!string.IsNullOrWhiteSpace(executingPath))
```

Comment:
`Assembly.Location` can never be null.

```suggestion
			if (executingPath.Length == 0)
```